assets/src/components/AssignmentTable.js

```diff
@@ -120,170 +144,271 @@ function AssignmentTable (props) {
     }
   }, [currentWeekRow.current])
 
+  const [assignmentGroupNames, setAssignmentGroupNames] = useState([])
+  useEffect(() => {
+    const allGroupNames = assignments.map(ag => ag.assignmentGroup.name)
+    var uniqueGroupNames = [...new Set(allGroupNames)]
+    uniqueGroupNames.sort()
+    setAssignmentGroupNames(uniqueGroupNames)
+  }, [assignments])
+
+  const [assignmentFilter, setAssignmentFilter] = useState('')
+  const handleNameFilterChange = e => {
+    setAssignmentFilter(e.target.value)
+  }
+
+  const ITEM_HEIGHT = 48
+  const ITEM_PADDING_TOP = 8
+  const MenuProps = {
+    PaperProps: {
+      style: {
+        maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+        width: 250
+      }
+    }
```

Comment:
I'm wondering if this could be defined in another file and imported - is there any other component that could potentially use `menuProps`?

Reply:
Currently I only see the Material Select being used in one other relevant place (ResourcesAccessed) but with no style properties defined.
I think it could make sense to put this somewhere common if we decide that we want this common appearance across more components but I feel like it might be early to know that is going to happen?